### DISCUSSION TITLE
Respawn setting not working on latest version 1.4.0.3

### DISCUSSION THREAD

**Discussion by IdioticRemi** [Pryaxis/TShock#1843] [Q&A]
I can't seem to get the respawn time to change from the default Terraria times. It used to work for some time but now it won't work anymore... Any clue what could be the problem?

**Comment by Buck39** [comment]
I also am seeing this happening.  No matter what I change in the config the default spawn time is 15 seconds with no boss present, even longer when there is one present.

**Reply by IdioticRemi** [reply]
Somehow it now works for me on a fresh installation done by hand (without docker)
